src/Features/Renderer.cpp

```diff
@@ -948,11 +952,23 @@ static bool SND_RecordBuffer_Hook(void)
 void Renderer::Frame()
 {
     if (Renderer::isDemoLoading && engine->hoststate->m_currentState == HS_RUN) {
-        Renderer::isDemoLoading = false;
         if (!g_render.isRendering.load() && sar_render_autostart.GetBool()) {
+
+            if (sar_render_autostart_odd_tick.GetBool()) {
+                // If the parity of the current tick does not match the parity of the first non-zero demo packet, don't start rendering
+                if (engine->demoplayer->GetTick() % 2 != Renderer::demoParity) {
```

Comment:
It might be worth checking the existing value here so we don't keep spamming alternateticks if playback is paused for whatever reason